scripts/linters/pylint_extensions.py

```diff
@@ -385,6 +407,102 @@ def check_functiondef_params(self, node, node_doc):
             node_doc, node.args, node,
             accept_no_param_doc=node_allow_no_param)
 
+    def check_arg_section_indentation(self, node):
+        """Checks whether the function argument definitions (Args: section) are
+        indented properly. Parameters should be indented by 4 relative to the
+        'Arg: ' line and any wrap-around in_descriptions should be indented by
+        8. The rest of the docstring section checks are in DocstringChecker.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+        """
+        arguments_node = node.args
+        expected_argument_names = set(
+            None if (arg.name in self.not_needed_param_in_docstring)
+            else (arg.name + ':') for arg in arguments_node.args)
+        currently_in_args_section = False
+        # When we are in the args section and a line ends in a colon,
+        # we can ignore the indentation styling in the next section of
+        # in_description, hence a freeform section.
```

Comment:
What's in_description? That concept doesn't seem to actually exist in this function, yet is referred to several times.

Reply:
This is something written by the previous implementer but I remembered you said that we generally don't use variables that start with is so since this is a boolean of the same kind of type(denotes whether or not we are parsing a docstring, I just changed it.

Reply:
But that was a different function entirely, right? In this function, that concept doesn't exist / hasn't been defined.